Modules/Loadable/Markups/MRMLDM/vtkMRMLMarkupsDisplayableManager.cxx

```diff
@@ -561,51 +553,36 @@ bool vtkMRMLMarkupsDisplayableManager::IsCorrectDisplayableManager()
 //---------------------------------------------------------------------------
 bool vtkMRMLMarkupsDisplayableManager::IsManageable(vtkMRMLNode* node)
 {
-  return (this->Focus.find(node->GetClassName()) != this->Focus.end());
+  if (node == nullptr)
+    {
+   vtkErrorMacro("Is Manageable: invalid node.");
+    }
+
+  return vtkMRMLMarkupsRegistrationFactory::GetInstance()->IsRegistered(node->GetClassName());
 }
 
 //---------------------------------------------------------------------------
 bool vtkMRMLMarkupsDisplayableManager::IsManageable(const char* nodeClassName)
 {
-  return nodeClassName && (this->Focus.find(nodeClassName) != this->Focus.end());
+  if (nodeClassName == nullptr)
+    {
+   return false;
```

Comment:
Indentation

Reply:
Thanks for the tip @cpinter, you are right.

Reply:
@cpinter, which tools do you use to check for code styling?